directives/point_of_interest.py

```diff
@@ -90,11 +89,23 @@ def run(self):
             # included in the A+ course materials (e-book).
             return []
 
-        # add an extra div to force content to desired height
-        hcontainer_opts = {
-            u'style':'height:'+self.options.get('height', '')+';',
-            u'class': container_class +' poi-content row',
-        }
+        if 'no_poi_box' not in self.options:
+            container_class = 'poi-container'
+            content_name = name + '-content'
+
+            # add an extra div to force content to desired height
+            hcontainer_opts = {
+                u'style': 'height:' + self.options.get('height', '') + ';',
+                u'class': container_class + ' poi-content row',
+            }
+        else:
+            container_class = 'no_poi_box'
```

Comment:
The naming style for CSS classes is to use hyphen `-`, not underscore `_`. So the class name becomes `no-poi-box`.

Reply:
I will fix this.